cudf/io.py

```diff
@@ -33,6 +33,8 @@ def read_csv(filepath, lineterminator='\n',
         Delimiter to be used.
     delim_whitespace : bool, default False
         Determines whether to use whitespace as delimiter.
+    sep : char, default None
+        Alternative argument name for delimiter. 
```

Comment:
To match Pandas as closely as possible, this should be:
```suggestion
sep : str, default ‘,’
    Delimiter to use.
```
And:

```suggestion
delimiter : str, default None
     Alternative argument name for sep.
```